sound/soc/intel/boards/skl_hda_dsp_generic.c

```diff
@@ -100,10 +100,10 @@ static int skl_hda_audio_probe(struct platform_device *pdev)
 	if (!card)
 		return -ENOMEM;
 
-	card->name = "hda-dsp",
-	card->owner = THIS_MODULE,
-	card->fully_routed = true,
-	card->late_probe = skl_hda_card_late_probe,
+	card->name = "hda-dsp";
+	card->owner = THIS_MODULE;
+	card->fully_routed = true;
+	card->late_probe = skl_hda_card_late_probe;
```

Comment:
Hmm, we need to add `Fixes: 33e59e50ee76 ("ASoC: Intel: skl_hda_dsp_generic: Allocate snd_soc_card dynamically")` in the commit message, and it is better to put the commit in the first commit. @ujfalusi FYI

Reply:
done

Reply:
Err...not sure how to do it...

Reply:
done. Thanks~